app/src/lib/editors/shared.ts

```diff
@@ -1,32 +1,11 @@
-import * as Darwin from './darwin'
-import * as Win32 from './win32'
-import * as Linux from './linux'
-
-export type ExternalEditor = Darwin.ExternalEditor | Win32.ExternalEditor
```

Comment:
Does removing `ExternalEditor` feel like we're losing _something valuable_?

As I mentioned in the PR description, it doesn't feel like we lost anything from a "compiler help" perspective. But maybe the code was clearer with it… maybe it's worth making `type ExternalEditor = string`? 🤔